students/17th/team-one/heeyoungjeong/user/views.py

```diff
@@ -0,0 +1,41 @@
+import json
+from django.http import JsonResponse, HttpResponse
+from django.views import View
+from json.decoder import  JSONDecodeError
+from user.models import User
+
+class SignUp(View):
```

Comment:
```suggestion
class SignUpView(View):
```
그냥 이름을 주기보다 View에 있는 클래스란 의미로 View를 붙여주는 네이밍을 권장 드립니다.
차후에 모델 클래스와 겹칠 가능성이 있기 때문에 애초에 분리하자는 의미로 그렇게 하고 있습니다.

Reply:
수정하였고 내용 숙지하겠습니다.